src/main/resources/swagger/api-docs.yaml

```diff
@@ -719,7 +722,19 @@ definitions:
         description: |
           Optional, the number of preemptible workers. If unspecified, the default number is 0. Ignored if numberOfWorkers is 0.
           For more information, see https://cloud.google.com/compute/docs/instances/preemptible
-
+  UserJupyterExtensionConfig:
+    description: 'Specification of Jupyter Extensions to be installed on the cluster'
+    properties:
+      nbExtensions:
+        type: object
+        description: |
+          Optional, map of extension name and notebook
```

Comment:
I think some more explanatory text and examples would be useful here. Maybe also mention that both .tar.gz files and python packages are supported.

Reply:
what about something like this:

'Optional, map of extension name and nbExtension. The nbExtension can either be a tar.gz file on google storage or a python package. Example, {"ext1":"gs:/bucket/extension.tar.gz", "pipExt":"python-packkage"}'

Reply:
Yes, that sounds good. 👍 

Similarly below for `serverExtensions`

Reply:
I like it, particularly the example . Note the typo in pack**k**age in case you copy/paste :)

Reply:
And gs:/*/*